Makefile

```diff
@@ -303,6 +303,12 @@ test-junit: build-kfctl
 	echo Running tests ... junit_file=$(JUNIT_FILE)
 	go test ./... -v 2>&1 | go-junit-report > $(JUNIT_FILE) --set-exit-code
 
+
```

Comment:
Clean-up target `test-junit` above, previously used by ksonnet.

Reply:
Addressed in latest commit